lib/manageiq/api/common/rbac/acls.rb

```diff
@@ -0,0 +1,74 @@
+module ManageIQ
+  module API
+    module Common
+      module RBAC
+        class ACLS
+          def create(resource_id, permissions)
+            permissions.collect do |permission|
+              create_acl(permission, resource_id)
+            end
+          end
+
+          def remove(acls, resource_id, permissions)
```

Comment:
It feels weird to me that acls is a parameter when you have a class that represents ACLs.  That is, the ACLs themselves should be part of the class and the methods act on it.  Or perhaps what would be nicer is an ACLCollection class that itself has the remove, add, etc. methods on it, with create returning an ACLCollection?

Reply:
(Doesn't have to be for this PR, but we might want to consider reorganizing as it's not very OO in design at the moment)

Reply:
Will change in the future if needs